spec/ParseUser.spec.js

```diff
@@ -3200,4 +3200,147 @@ describe('Parse.User testing', () => {
       .then(done)
       .catch(done.fail);
   });
+
+  it('can login with email', (done) => {
+    const user = new Parse.User();
+    user.save({
+      username: 'yolo',
+      password: 'yolopass',
+      email: 'yo@lo.com'
+    }).then(() => {
+      const options = {
+        url: `http://localhost:8378/1/login`,
+        headers: {
+          'X-Parse-Application-Id': Parse.applicationId,
+          'X-Parse-REST-API-Key': 'rest',
+        },
+        json: { email: "yo@lo.com", password: 'yolopass'}
+      }
+      return rp.get(options);
+    }).then(done).catch(done.fail);
+  });
+
```

Comment:
Could have an additional test here for trying `cannot login with email and bad password`, just to cover if a possible bug were introduced regarding password validation with an email at any point.

Reply:
Added the test on the next line.